src/OpenSSL/rand.py

```diff
@@ -2,8 +2,16 @@
 PRNG management routines, thin wrappers.
 """
 
+import warnings
+
 from OpenSSL._util import lib as _lib
 
+warnings.warn(
+    "OpenSSL.rand is deprecated - you should use os.urandom instead",
+    DeprecationWarning,
+    stacklevel=3,
```

Comment:
Stacklevel 3 because of how CPython does imports?

Reply:
Because that's what I used the last time I tried to deprecate this 😭

Reply:
🥲